packages/settings-library/src/settings_library/celery.py

```diff
@@ -0,0 +1,52 @@
+from datetime import timedelta
+from typing import Annotated
+
+from pydantic import Field
+from pydantic_settings import SettingsConfigDict
+from settings_library.rabbit import RabbitSettings
+from settings_library.redis import RedisSettings
+
+from .base import BaseCustomSettings
+
+
+class CelerySettings(BaseCustomSettings):
+    CELERY_BROKER: Annotated[
+        RabbitSettings, Field(json_schema_extra={"auto_default_from_env": True})
+    ]
+    CELERY_RESULT_BACKEND: Annotated[
+        RedisSettings, Field(json_schema_extra={"auto_default_from_env": True})
```

Comment:
I would suggest to call this simply `CELERY_REDIS_BACKEND`, even if multiple backends are possible in theory. It is good to see on first glance what this setting expects.